app.py

```diff
@@ -1,5 +1,10 @@
 from api.views import app
+from api.dbconnect import Dbconnection
+
+testss = Dbconnection()
```

Comment:
Consider changing object name to reflect relevance to class. For instance, `dbconnect=Dbconnection()`